docs/src/pages/index.tsx

```diff
@@ -1,12 +1,22 @@
 // REACT
 import * as React from 'react';
 // VENDOR
-import styled, { css } from 'styled-components';
+import styled, { createGlobalStyle, css } from 'styled-components';
 import { withPrefix } from 'gatsby';
 // COMPONENTS
 import { Page } from '../components/Layout';
 // THEME
 
+const IndexPageGlobalCSS = createGlobalStyle`
+  html,
```

Comment:
This is a little alarming. What's this thing for?

Reply:
@sharkparty I moved those styles from `src/theme/GlobalStyles/Globals.styles.ts` to here because it seemed that they were necessary only on the index page (they seem to be for the wave animation).

Reply:
We should try to make them only impact this page and remove the global override, or leave them in there until we remove the splash page. There are definitely things in the globals just for the splash page, like that bright red BG, but I don't want to globalize that better, I want that entry to be refactored when we land the new index page.